app/views/notifications/_notifications_list.html.erb

```diff
@@ -30,3 +17,10 @@
     If you see too many of these, please report it to <a href="mailto:yo@dev.to">yo@dev.to</a>!
   </div>
 <% end %>
+
+<% if @notifications.any? %>
```

Comment:
this was good where it was in the beginning, after the block of notifications :)